### PR TITLE
[mod_articles_popular]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_articles_popular
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_articles_popular/mod_articles_popular.xml [hunk 1]
```diff
@@ -40,74 +40,84 @@
...
 					size="22"
 					filter="user_utc"
-					showon="date_filtering:range" />
+					showon="date_filtering:range"
+					/>
 
-				<field name="relative_date" type="text" default="30"
+				<field
+					name="relative_date"
+					type="text" default="30"
```
Comment (andrepereiradasilva) [inline modules/mod_articles_popular/mod_articles_popular.xml]: move default down?

File: modules/mod_articles_popular/mod_articles_popular.xml [hunk 2]
```diff
@@ -116,35 +126,36 @@
...
+					default="900"
+				/>
 
 				<field
 					name="cachemode"
 					type="hidden"
 					default="static">
-					<option
-						value="static"></option>
+					<option value="static"></option>
```
Comment (andrepereiradasilva) [inline modules/mod_articles_popular/mod_articles_popular.xml]: move > to a new line?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 40b81048f3e4f2266bf1b3d520e388226a74633a<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10576">issues.joomla.org/joomla-cms/10576</a>.</sub>

File: modules/mod_articles_popular/mod_articles_popular.xml [hunk 3]
```diff
@@ -40,74 +40,85 @@
...
+					description="MOD_POPULAR_FIELD_RELATIVEDATE_DESC"
+					default="30"
+					showon="date_filtering:relative"
+				/>
 			</fieldset>
 
-			<fieldset
-				name="advanced">
-
+			<fieldsetname="advanced">
```
Comment (JoomliC) [inline modules/mod_articles_popular/mod_articles_popular.xml]: missing space between fieldset and name ...

**Comment by JoomliC** [comment]
I have tested this item :red_circle: unsuccessfully on 40b81048f3e4f2266bf1b3d520e388226a74633a<br /><br />Error here: https://github.com/joomla/joomla-cms/pull/10576/files#diff-a116326b5f017de7f20b52ba1620c439R121<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10576">issues.joomla.org/joomla-cms/10576</a>.</sub>

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 2bad85aa47b7064e49b708d835fb7b6af572177d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10576">issues.joomla.org/joomla-cms/10576</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 2bad85aa47b7064e49b708d835fb7b6af572177d<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10576">issues.joomla.org/joomla-cms/10576</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10576">issues.joomla.org/joomla-cms/10576</a>.</sub>

**Pull request closed without merging**